source/documentation/common/upgrade/snip-upgrade_considerations.adoc

```diff
@@ -2,4 +2,15 @@
 :_content-type: SNIPPET
 .Upgrade Considerations
 
-When planning to upgrade, see link:https://access.redhat.com/articles/5268351[Red Hat Virtualization 4.4 upgrade considerations and known issues].
+* When planning to upgrade, see link:https://access.redhat.com/articles/5268351[Red Hat Virtualization 4.4 upgrade considerations and known issues].
+
+* When upgrading from Open Virtual Network (OVN) and Open vSwitch (OvS) 2.11 to OVN 2021 and OvS 2.15, the process is transparent to the user as long as the following conditions are met:
+
+** The engine is upgraded first.
+** The ovirt-provider-ovn security groups must be disabled, before the host upgrade, for all OVN networks that are expected to work between hosts with OVN/OvS version 2.11.
+** The hosts are upgraded to match OVN version 2021 or higher and OvS version 2.15.
+
```

Comment:
I would use the doc text from the RFE bug:
Upgrade from OvS/OVN 2.11 to OVN 2021 and OvS 2.15.

The upgrade is transparent to user as long as few conditions are met:

1) The engine must be upgraded first.
2) Before any host upgrade the ovirt-provider-ovn security groups must be disabled for all OVN networks that are expected to work between host with OVN/OvS version 2.11.
3) Upgrade hosts to match the OVN version 2021 or higher and OvS version to 2.15. This step should be done via the UI, in order to reconfigure OVN and refresh certificates.
4) Reboot the host after upgrade


To check if provider and OVN was configured successfully on the host, webadmin includes OVN configured flag in General tab for every host, at the same time this value is available via REST API. Please note that this value might say "No" until capabilities are refreshed for the host.

Fixing the host's OVN not being configured can be done by reinstalling it from engine 4.5 or higher.

Reply:
Looks goof to me. Small note, maybe it's worth to highlight that the host upgrade via the web admin portal is a MUST step in order the ovn and certificates properly reconfigured.

Thank you